docs/maintainers/releasing.md

```diff
@@ -2,23 +2,34 @@
 
 ## Release Steps
 
-> Note: Step 2, 3 and 4 can be done in parallel. Step 5 and 6 can be done in parallel.
+When creating a release milestone, you should send meeting invites to maintainers to book the release day and the previous day.
+This is to make sure they have enough time to work on the release.
 
-1. Create a branch named `release` in `PowerShell/PowerShell` repository. All release related changes should happen in this branch.
+The following release procedure should be started on the previous day of the target release day.
+This is to make sure we have enough buffer time to publish the release on the target day.
+
+Before starting the following release procedure, open an issue and list all those steps as to-do tasks.
```

Comment:
Perhaps make this a to-do list and number the step explicitly inside the todo.  That way you can just copy the markdown into the issue?